etc/config/crystal.amazon.properties

```diff
@@ -0,0 +1,38 @@
+compilers=&crystal
+defaultCompiler=crystal10
+
+group.crystal.compilers=crystal10:crystal036:crystal035:crystal034:crystal033:crystal032:crystal031:crystal030:crystal029
+group.crystal.isSemVer=true
+group.crystal.groupName=Crystal x86-64
+group.crystal.supportsBinary=true
+group.crystal.supportsExecute=true
+group.crystal.compilerType=crystal
+group.crystal.objdumper=/opt/compiler-explorer/gcc-snapshot/bin/objdump
+
+compiler.crystal10.name=Crystal 1.0
+compiler.crystal10.semver=1.0.0
+compiler.crystal10.exe=/opt/compiler-explorer/crystal-1.0.0/bin/crystal
```

Comment:
Argh! I did not properly send the review!
This would be better if you added a
`group.crystal.baseName=Crystal` property.

That way, you don't have to specify a `.name` in each compiler, and instead, the final displayed one will be
the combination of `baseName` and each compiler's `semver`.